corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -6,11 +6,91 @@
 
 {% block page_navigation %}
   <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
-  <ul class="nav nav-hq-sidebar" data-bind="foreach: caseTypes">
+  <ul class="nav nav-hq-sidebar">
+    <!-- ko foreach: caseTypes -->
```

Comment:
Just curious: do you like the knockout comment syntax? I usually only see it in our code for performance reasons (to avoid adding an element that has no purpose other than having a binding attached).

Reply:
I do not generally like it but the alternative was to add a div solely for the purpose of binding the foreach

Reply:
Can't the foreach be bound to the ul?

Reply:
no because I'm adding in a static `<li>` after the loop but still inside the `<ul>`